app/app-config.production.yaml

```diff
@@ -104,9 +104,9 @@ catalog:
         clientId: ${AUTH_MICROSOFT_CLIENT_ID}
         clientSecret: ${AUTH_MICROSOFT_CLIENT_SECRET}
         userGroupMember:
-          filter: "displayName eq '${ADP_PORTAL_PLATFORM_ADMINS_GROUP}' or displayName eq '${ADP_PORTAL_PROGRAMME_ADMINS_GROUP}'"
+          filter: "displayName eq '${ADP_PORTAL_PLATFORM_ADMINS_GROUP}' or startsWith(displayName, '${ADP_PORTAL_USERS_GROUP_PREFIX}')"
```

Comment:
Do we want to include users with no related roles too, so that project admins can add new people to their project? If someone hasnt been added to a project before they wont be in any of the related groups

Reply:
Anyone who is not in any of these AD groups won't be able to access the Portal. So the question is should we import all users upfront. I would say it requires some further discussion. Can this PR go in as is, and we have a separate discussion on importing all users.